src/guide/1/geofencing/javascript/snippets/geofenc.js

```diff
@@ -0,0 +1,63 @@
+const {
+  Kuzzle,
+  WebSocket
+} = require('kuzzle-sdk');
+const kuzzle = new Kuzzle(new WebSocket('kuzzle'));
+const bigBen = {
+  lat: 51.510357,
+  lon: -0.116773
+};
+// Create a filter that defines the circular area around Big Ben
+const filter = {
+  geoDistance: {
+    location: bigBen,
+    distance: '2km'
+  }
+};
+// Create the user's location: they are inside the circular area
+const currentLocation = {
+  firstName: 'Ada',
+  lastName: 'Lovelace',
+  location: bigBen
+};
+const hydePark = {
+  lat: 11.507268,
+  lon: -0.165730
+};
+const newLocation = {location: hydePark};
+const run = async () => {
+  const callback = async (notification) => {
+    if (notification.scope === 'out') {
```

Comment:
I think you meant `scope === 'in'`:

```suggestion
    if (notification.scope === 'in') {
```

If you did want to use the `out` value, then you should print the following on the console instead:

```js
console.log('User has left Big Ben');
```

Another alternative: use both cases, and test both console values :wink: